client/README.md

```diff
@@ -0,0 +1,36 @@
+<img src="images/gobra.png" height="250">
+
+Gobra IDE directly integrates verification with Gobra into VSCode.
+
```

Comment:
Is this file shown to users of the IDE? In that case, someone with better English abilities than me should go over this file once. Maybe you can ask Wytse or Martin.

Reply:
Furthermore, at some point we should absolutely unambiguously that this is a prototype or an experimental tool.

Reply:
Yes it will.
I've added the following info as second sentence:
> **Note:** Gobra as well as Gobra IDE are both prototypes and under active development.